source/server/hot_restarting_parent.cc

```diff
@@ -108,7 +108,7 @@ HotRestartingParent::Internal::getListenSocketsForChild(const HotRestartMessage:
     Network::ListenSocketFactory& socket_factory = listener.get().listenSocketFactory();
     if (*socket_factory.localAddress() == *addr && listener.get().bindToPort()) {
       if (socket_factory.sharedSocket().has_value()) {
-        // Pass the socket to the new process iff it is already shared across workers.
+        // Pass the socket to the new process if it is already shared across workers.
```

Comment:
This is not actually a typo (if and only if) but it's OK to change.

Reply:
Got it. Thanks.